packages/components/src/internal/components/navigation/model.ts

```diff
@@ -16,11 +16,9 @@
 import { List, Record } from 'immutable';
 import { ActionURL, Ajax, Utils, QueryKey } from '@labkey/api';
 
-import { AppURL } from '../../url/AppURL';
+import { createProductUrl, createProductUrlFromParts, AppURL } from '../../url/AppURL';
```

Comment:
Just curious, should all of these imports, including `buildURL`, be switched to `'../..'` (...may be more dots)?